packages/perspective-viewer-d3fc/src/js/plugin/template.js

```diff
@@ -72,6 +72,15 @@ class D3FCChartElement extends HTMLElement {
         return this._container;
     }
 
+    getSettings() {
+        return this._settings;

```

Comment:
I wonder if we should make sure we *only* save the user settings - i.e. don't save or restore the config that we get anyway via the `draw` method (row and column pivots, filters etc).

Reply:
Yes, I agree. This was just an initial look but I think we need to decide what actually needs to be saved.